tracing/suspend_resume_time.bt

```diff
@@ -0,0 +1,24 @@
+// This tracks the time it takes for runtime and system suspend/resume
```

Comment:
// SPDX-License-Identifier: BSD-3-Clause
// Copyright(c) 2022 Intel Corporation. All rights reserved.

Reply:
None of the files in sof-test had this, are you sure it's necessary?

Reply:
I prefer to avoid any ambiguity related to license/copyright.